test/e2e/node/pod_resize.go

```diff
@@ -19,6 +19,8 @@ package node
 import (
 	"context"
 	"fmt"
+	resourceapi "k8s.io/kubernetes/pkg/api/v1/resource"
```

Comment:
same here. please move resourceapi import to the line below kubecm
and e2ekubelet to the line between "k8s.io/kubernetes/test/e2e/framework" and e2epod "k8s.io/kubernetes/test/e2e/framework/pod"